storybook/pages/StatusChatInputPage.qml

```diff
@@ -90,8 +92,22 @@ SplitView {
                 usersStore: QtObject {
                     readonly property var usersModel: fakeUsersModel
                 }
+                onSendMessage: {
+                    logs.logEvent("StatusChatInput::sendMessage", ["MessageWithPk"], [chatInput.getTextWithPublicKeys()])
+                    logs.logEvent("StatusChatInput::sendMessage", ["PlainText"], [globalUtilsMock.globalUtils.plainText(chatInput.getTextWithPublicKeys())])
+                    logs.logEvent("StatusChatInput::sendMessage", ["RawText"], [chatInput.textInput.text])
+                }
```

Comment:
Do we want to keep these around?

Reply:
Not necessarily. I can remove them if you think it's polluting.
Since sendMessage is an important part of StatusChatInput I thought it might be worth having an idea on what the message will look like in its different forms while working on Storybook.

Reply:
Ah sorry, I totally missed that this is inside storybook. Keep them